components/account/AccountHeader.vue

```diff
@@ -107,7 +121,11 @@ const isNotifiedOnPost = $computed(() => !!relationship?.notifying)
             </NuxtLink>
             <AccountFollowButton :account="account" :command="command" />
             <span inset-ie-0 flex gap-2 items-center>
-              <AccountMoreButton :account="account" :command="command" />
+              <AccountMoreButton
+                :account="account" :command="command"
+                @add-note="isEditingPersonalNote = true"
+                @remove-note="isEditingPersonalNote = false"
```

Comment:
There isn't a `remove-note` event. Probably this stayed after a refactoring. I wonder if you could also handle the `add-note` directly in the AccountMoreButton.
Apart from that, this looks great 👍🏼